pkg/proxy/ipvs/util/ipvs_linux.go

```diff
@@ -27,6 +27,7 @@ import (
 	"time"
 
 	"errors"
+
```

Comment:
Remove the empty line above "errors". It is part of the standard lib

Reply:
(and re-alphabetize)